### ISSUE TITLE
monitor scrape is timeout in my cluster ，over 30000 pods

### ISSUE THREAD

**Issue by for800000** [Project-HAMi/HAMi#672]
**Please provide an in-depth description of the question you have**:
curl metrics port need more than 30s
**What do you think about this question?**:
decrease monitor time-consuming
**Environment**:
- HAMi version:
v2.4.0
- Kubernetes version:
v.1.22
- Others:
![image](https://github.com/user-attachments/assets/a2f6cbe4-468c-4aab-8900-2715af781fde)
![image](https://github.com/user-attachments/assets/b3005ad0-d0fa-4bb3-899e-f79514c5aa1b)
![image](https://github.com/user-attachments/assets/0cc98a11-d015-4541-b3b8-312e39d18c14)

**Comment by archlitchi** [comment]
which monitor? 31993 or 31992?

**Comment by for800000** [comment]
31992

**Comment by archlitchi** [comment]
got it, i see you already have a solution, perhaps you would like to submit a PR?

**Comment by for800000** [comment]
yes, is there any problem https://github.com/Project-HAMi/HAMi/pull/683

**Comment by for800000** [comment]
@archlitchi i'm sorry to losing this chart yaml,what should i do？
![image](https://github.com/user-attachments/assets/14cda59e-577b-46ea-b149-d1a219082045)
